### PR TITLE
chore: update dependencies

### PR DESCRIPTION
**Pull request by frytg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +13,17 @@ ARD system to distribute real-time (live) metadata for primarily radio broadcast
   - [Third-Party Components](#third-party-components)
   - [Authors](#authors)
 
+## Active Data Suppliers
+
+These ARD broadcasters are currently sending live metadata via ARD Eventhub:
+
+| Broadcaster | TEST | PROD |
+| ----------- | ---- | ---- |
+| mdr         | ✅   | ✅   |
```
Comment (rafaelmaeuer) [inline README.md:22]: is mdr be written lowercase?

File: package.json [hunk 2]
```diff
@@ -13,6 +13,7 @@
 		"ingest:local": "nodemon -r dotenv/config ./src/ingest/index.js",
 		"ingest:cloud": "node -r dotenv/config ./src/ingest/index.js",
 		"license": "chmod +x license.sh && ./license.sh",
+		"reinstall": "rm -rf node_modules && rm yarn.lock && yarn",
```
Comment (rafaelmaeuer) [inline package.json:16]: ❤️

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "ard-eventhub",
-	"version": "1.2.3",
+	"version": "1.2.4",
```
Comment (rafaelmaeuer) [inline package.json:3]: please change version in `openapi.json` and `openapi.yml` too!

**Review by rafaelmaeuer — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (frytg) [reply README.md:22]: yes

**Review by rafaelmaeuer — APPROVED** [review]

**Pull request merged**